pydra/compose/base/helpers.py

```diff
@@ -66,7 +66,7 @@ def ensure_field_objects(
                 )
             else:
                 arg.name = input_name
-            if not arg.help:
+            if not arg.help and input_helps:
```

Comment:
If `input_helps` is `{}`, `arg.help` will no longer go from `None` or something Falsey to the specific value `""`. Is that okay?

Reply:
Not sure, but will change so it does